kernel/model/selector.go

```diff
@@ -85,6 +85,17 @@ func (m *Model) MustSelectRegion(regionSpec string) *Region {
 	return region
 }
 
+func (m *Model) GetRegion(host *Host) *Region {
+	for _, region := range m.Regions {
+		for _, rh := range region.Hosts {
+			if host == rh {
+				return region
+			}
+		}
+	}
+	panic(errors.Errorf("region for host %+v not found", host))
+}
```

Comment:
This gives me feelings.

It looks like this is here just to emit the region when doing the new `fablab ls hosts`. Traditionally, the way I go after host information is `fablab dump`. Wouldn't it be better just to iterate regions in your list command, and then emit the hosts in region order, rather that doing this?

Reply:
I'll admit I knew this was likely to cause you mental anguish and left it there to see if you had any better ideas :)

I didn't go down the route you suggest because I didn't want to re-implement the region selection logic. However thinking about it now, I can make a SelectHosts on Region, then use than in model.SelectHosts and also in list.

Reply:
So you're torturing me with PRs. Got it. 🤣